tutorial/index.html.script

```diff
@@ -1512,6 +1538,120 @@ object SharedJournalHelper {
 </div>
 
 <div>
+    <h2>Deploying Reactive Applications</h2>
+
+    <p>
+      We have developed <a href="https://www.typesafe.com/products/conductr" target="_blank">ConductR</a>;
+      a solution that is designed to ease the deployment and
+      management of reactive applications and services. ConductR handles failures gracefully, scales elastically and
+      embraces change in your Reactive system. Enough of the sales talk though, let's ship this app!
+    </p>
+
+    <p>
+      For any application that will target ConductR you first need to bring in
+      <a href="#code/project/plugins.sbt" class="shortcut">a couple of plugins</a>, namely
+      <a href="https://github.com/sbt/sbt-conductr" target="_blank">sbt-conductr</a> and
+      <a href="https://github.com/typesafehub/sbt-conductr-sandbox" target="_blank">sbt-conductr-sandbox</a>. You will also need to have installed
+      <a href="https://www.docker.com/" target="_blank">Docker</a> in order to run ConductR within the
+      sandbox.
+    </p>
+
+    <p>
+      sbt-conductr actually brings in another plugin named <a href="https://github.com/sbt/sbt-bundle" target="_blank">sbt-bundle</a> and between them, you declare some
+      <a href="#code/build.sbt" class="shortcut">settings</a> required for deploying your application.
+      You then load and run a "bundle" which is the
+      unit of deployment for ConductR. A bundle is a zip file with the hash of its contents encoded in its name.
+      Think of the hash as a verifiable version number. When the zip file's contents changes then the
+      hash will change as well. ConductR verifies this hash thus providing the confidence that version xyz
+      is absolutely version xyz and has not been tampered with in any way. Given the uniqueness of this hash, you
+      can support rolling upgrades by having multiple versions of your application running at the same time. You
+      also know what point to rollback to should an upgrade of your application not go the way you planned.
```

Comment:
This feels to me like too much detail for the reactive maps tutorial while at the same time not being enough detail to adequately explain the importance and motivations behind bundle hashes.  Perhaps just mentioning that a bundle is identified by its hash, and a link to the ConductR documentation on bundle hashes would be better?